src/test/java/org/eclipse/microprofile/starter/core/artifacts/DirectoryCreatorTest.java

```diff
@@ -0,0 +1,22 @@
+package org.eclipse.microprofile.starter.core.artifacts;
+
+import org.eclipse.microprofile.starter.FakeDirectoryCreator;
+import org.eclipse.microprofile.starter.core.model.JessieMaven;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
CheckStyle fails on star imports.

instead of `assertEquals` you can use `Assert.assertEquals` and use a regular import